src/bundlers/node/lambdaHandlerGenerator.ts

```diff
@@ -1,6 +1,8 @@
 //const object = new handler.genezio[Object.keys(handler.genezio)[0]]();
 /* eslint-disable no-useless-escape */
+import getUser from "../../requests/getUser.js";
 
+const user = await getUser();
```

Comment:
this call should be made inside a function upper level and only send the timeout as a param to the `lambdaHandlerGenerator` function to keep it more dynamic